samples/net/rpl-node/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _rpl-node-sample:
+
+RPL node
+###########
+
+Overview
+********
+
+A simple RPL node showing how to join in RPL mesh network.
+
+This demo assumes that the platform of choice has networking support,
+some adjustments to the configuration may be needed.
+
+The sample will listen for RPL multicast messages and joins with RPL
+Border Router node in DAG network. It exposes few resources through
+CoAP server role.
+
+The sample exports the following resources:
+
+.. code-block:: none
+
+   /led
+   /ipv6/neighbors
+   /rpl-info
+   /rpl-info/parent
+   /rpl-info/rank
+   /rpl-info/link-metric
+
+These resources allow to toggle led on board (if boards supports) and build
+rpl mesh network topology from node rpl information.
+
+Building And Running
+********************
+
+If border router is a Sparrow border router, follow below steps to build and
```

Comment:
Change this line to:

     If you're using a Sparrow border router, follow the steps below to build and